src/App.js

```diff
@@ -1,29 +1,22 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+// eslint-disable-next-line
```

Comment:
- [ ] it is always a good practice to make sure you fix manually all linters errors instead of disabling them, A linter is great for identifying errors when you use standard rules,  Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
![image](https://user-images.githubusercontent.com/80157011/176185449-c5a9f449-e816-4d86-8954-65987427033d.png)

As you can see when I use the linters without those comments I have errors because for react is preferred to utilize function components instead of class components.
Since it is a core requirement for this project to use class components I have no other option than keep those disabling comments.